roles/kubernetes/control-plane/defaults/main/main.yml

```diff
@@ -18,6 +18,11 @@ kube_apiserver_node_port_range: "30000-32767"
 # ETCD backend for k8s data
 kube_apiserver_storage_backend: etcd3
 
+# CIS 1.2.26
+# Validate that the service account token
+# in the request is actually present in etcd.
+kube_apiserver_service_account_lookup: true
```

Comment:
Is there an usecase where this would need to be set to false? If no, then why not just hardcode the proper value in kubeadm-config ?

Reply:
Just in case you want to disable the service account validation.

Reply:
I gather that but why whould one want to disable service account validation?

Reply:
I guess if service-account-lookup feature affects some performance due to looking up on etcd, some operators might want to disable it.
As https://www.tenable.com/audits/items/CIS_Kubernetes_v1.20_v1.0.0_Level_1_Master.audit:d28b52afd28881a177809af260ca96bd the default value is true on Kubernetes side.
So I feel there is not so big merit of this additional parameter at this time from security viewpoint at this time.
On the other hand, I think there is not strong reason why we need to drop this pull request.
We can clarify service-account-lookup is enabled on Kubespray side with this pull request and we can avoid the change of the default value if that happens on Kubernetes side.

Reply:
Exactly, this is not strongly needed for security but it's useful to make "manageable" the parameter and keep it with IaC.
What I mean is that in case the parameter should switch to `false` for some reason, having a `hardening.yaml` (extra vars file) with all the configuration to setup the cluster, we can not worry about this because the parameter is manageable and the file set it automatically to `true`.